src/Security/test/AuthSamples.FunctionalTests/AuthSamples.FunctionalTests.csproj

```diff
@@ -6,6 +6,10 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
This is just for testing purposes. Need to actually identify a list of tests that need to be run on OSX 10.14